scripts/output/single/ar6Climate.R

```diff
@@ -0,0 +1,228 @@
+remindReportingFile   <- file.path(outputdir, paste0("REMIND_generic_", scenario, ".mif"))
+climateAssessmentEmi  <- normalizePath(file.path(outputdir, paste0("ar6_climate_assessment_", scenario, ".csv")),
+                                       mustWork = FALSE)
+climateAssessmentYaml <- file.path(system.file(package = "piamInterfaces"),
+                                   "iiasaTemplates", "climate_assessment_variables.yaml")
+
+############################# PREPARING EMISSIONS INPUT #############################
+
+logmsg <- paste0(
+  date(), " =================== CONFIGURATION STARTED ==================================\n",
+  "  outputdir = '",             outputdir,              "' exists? ", file.exists(outputdir), "\n",
+  "  cfgPath = '",               cfgPath,                "' exists? ", file.exists(cfgPath), "\n",
+  "  logFile = '",               logFile,                "' exists? ", file.exists(logFile), "\n",
+  "  scenario = '",              scenario, "'\n",
+  "  remindReportingFile = '",   remindReportingFile,   "' exists? ", file.exists(remindReportingFile), "\n",
+  "  climateAssessmentYaml = '", climateAssessmentYaml, "' exists? ", file.exists(climateAssessmentYaml), "\n",
+  "  climateAssessmentEmi = '",  climateAssessmentEmi,  "' exists? ", file.exists(climateAssessmentEmi), "\n",
+  date(), " =================== EXTRACT REMIND emission data ===========================\n",
+  "  ar6Climate.R: Extracting REMIND emission data\n"
+)
+cat(logmsg)
+capture.output(cat(logmsg), file = logFile, append = TRUE)
+
+climateAssessmentInputData <- as.quitte(remindReportingFile) %>%
+  # Consider only the global region
+  filter(region %in% c("GLO", "World")) %>%
+  # Extract only the variables needed for climate-assessment. These are provided from the iiasaTemplates in the
+  # piamInterfaces package. See also:
+  # https://github.com/pik-piam/piamInterfaces/blob/master/inst/iiasaTemplates/climate_assessment_variables.yaml
+  generateIIASASubmission(
+    mapping = "AR6",
+    outputFilename = NULL,
+    iiasatemplate = climateAssessmentYaml,
+    logFile = logFile
+  ) %>%
+  mutate(region = factor("World")) %>%
+  # Rename the columns using str_to_title which capitalizes the first letter of each word
+  rename_with(str_to_title) %>%
+  # Transforms the yearly values for each variable from a long to a wide format. The resulting data frame then has
+  # one column for each year and one row for each variable
+  pivot_wider(names_from = "Period", values_from = "Value") %>%
+  write_csv(climateAssessmentEmi, quote = "none")
+
+logmsg <- paste0(
+  date(), "  ar6Climate.R: Wrote REMIND emission data to '", climateAssessmentEmi, "' for climate-assessment\n"
+)
+cat(logmsg)
+capture.output(cat(logmsg), file = logFile, append = TRUE)
+
+############################# PYTHON/MAGICC SETUP #############################
+
+# Read the cfg to get the location of MAGICC-related files
+cfg <- read_yaml(cfgPath)
```

Comment:
Ok to add that? People will definitely want to run that on old runs.
```suggestion
cfg <- read_yaml(cfgPath)

if (is.null(cfg$climate_assessment_root)) cfg$climate_assessment_root <- "/p/projects/rd3mod/python/climate-assessment/src/"
if (is.null(cfg$climate_assessment_files_dir)) cfg$climate_assessment_files_dir <- "/p/projects/rd3mod/climate-assessment-files/"
if (is.null(cfg$cfg$climate_assessment_magicc_bin)) cfg$climate_assessment_magicc_bin <- "/p/projects/rd3mod/climate-assessment-files/magicc-v7.5.3/bin/magicc"
```